Makefile

```diff
@@ -22,9 +22,13 @@ build: build-go
 build-go:
 	go build ./...
 
+.PHONY: rm-cli
+rm-cli:
+	rm -f $(BIN_OUTPUT_PATH)/viam-cli
+
 GOOS ?= $(shell go env GOOS)
 GOARCH ?= $(shell go env GOARCH)
-bin/$(GOOS)-$(GOARCH)/viam-cli:
+bin/$(GOOS)-$(GOARCH)/viam-cli: rm-cli
```

Comment:
This is an interesting consequence of using `make` is a "front-end" for `go build`. Make can't tell whether a dependency has changed in this case